src/lib/components/forms/form/form-components/text-area/text-area.component.spec.ts

```diff
@@ -81,6 +81,68 @@ describe('TextAreaComponent', () => {
         expect(await input.isDisabled()).toBe(true);
     });
 
+    it('should only accept #maxLength amount of characters', async () => {
+        const maxLength: number = 10;
+        const validString: string = new Array(maxLength).fill('x').join('');
+        const invalidString: string = new Array(maxLength + 1).fill('x').join('');
+        component.maxLength = maxLength;
+
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        await input.setValue(validString);
+        expect(await formField.isControlValid()).toBe(true);
+        await input.setValue(invalidString);
+        expect(await formField.isControlValid()).toBe(false);
+    });
+
+    it('should require #minLength amount of characters', async () => {
+        const minLength: number = 5;
+        const validString: string = new Array(minLength).fill('x').join('');
+        const invalidString: string = new Array(minLength - 1).fill('x').join('');
+        component.minLength = minLength;
+
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        await input.setValue(validString);
+        expect(await formField.isControlValid()).toBe(true);
+        await input.setValue(invalidString);
+        expect(await formField.isControlValid()).toBe(false);
+    });
+
+    it('should have the correct error message for minLength ', async () => {
+        const minLength: number = 5;
+        const invalidString: string = Array(minLength - 1)
+            .fill('x')
+            .join('');
+
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        component.minLength = minLength;
+        await input.setValue(invalidString);
+        await input.blur();
+        fixture.detectChanges();
+        expect(await formField.hasErrors()).toBeTrue();
+
+        const errors: Array<string> = await formField.getTextErrors();
+        expect(errors.length).toBe(1);
+        expect(errors[0]).toBe('terraTextInput.minLength');
```

Comment:
```suggestion
        expect(errors.includes('terraTextInput.minLength')).toBe('true);
```
dann kannst du dir oben das length sparen